core/shared/src/main/scala/monocle/function/At.scala

```diff
@@ -21,6 +23,36 @@ abstract class At[S, I, A] extends Serializable {
 trait AtFunctions {
   def at[S, I, A](i: I)(implicit ev: At[S, I, A]): Lens[S, A] = ev.at(i)
 
+  /**
+    * Creates a Lens that zooms into an index `i` inside `S`.
```

Comment:
It really is a pity that there isn't a better way to link documentation in scala. These examples feel as though they belong elsewhere.